### DISCUSSION TITLE
Welcome to team-compass Discussions!

### DISCUSSION THREAD

**Discussion by choldgraf** [2i2c-org/team-compass#78] [General]
## 👋 Welcome!

This is a place to discuss team-wide topics across 2i2c.

**Comment by damianavila** [comment]
Super! So the idea is to actually start discussions here to shape non-ready deliverables among other things? Or are you thinking of this space as a broader space to deal with other sorts of team discussions?

**Reply by choldgraf** [reply]
I imagined that we'd use it for discussions that aren't related directly to the `pilot-hubs` repository (since we have https://github.com/2i2c-org/pilot-hubs/discussions), and we can use the `meta/` Discussions space for topics that are sensitive or shouldn't be in public.

Do you think that this is too many Discussions locations? I worry we are fracturing a bit too much too early, but it felt like `pilot-hubs` was a bit too specific for team-wide discussion

**Reply by damianavila** [reply]
I think 3-4 places is probably enough for a starting point as long as there is someplace (probably the team compass docs) collecting the links to those places. In fact, any time you a well-defined niche, you can start a specific discussion place as long as you signal to others where that place lives, IMHO.

**Reply by choldgraf** [reply]
So in my mind the niche is:

- Discussion specifically about the pilot hubs: https://github.com/2i2c-org/pilot-hubs/discussions
- Team discussion that can be public: https://github.com/2i2c-org/team-compass/discussions
- Team discussion that should be private (probably low-volume): https://github.com/2i2c-org/meta/discussions

Is that a reasonable breakdown?

**Reply by damianavila** [reply]
I think it is reasonable! Let's just document that somewhere in the team compass, IMHO.

**Reply by choldgraf** [reply]
👉  https://github.com/2i2c-org/team-compass/pull/84
